src/sbvr-api/sbvr-utils.ts

```diff
@@ -1192,11 +1196,103 @@ export const getModel = (vocabulary: string) => {
 	return models[vocabulary];
 };
 
+const validateBatch = (req: Express.Request) => {
+	const { requests } = req.body as { requests: uriParser.UnparsedRequest[] };
+	if (!Array.isArray(requests)) {
+		throw new BadRequestError(
+			'Batch requests must include an array of requests in the body via the "requests" property',
+		);
+	}
+	if (req.headers != null && req.headers['content-type'] == null) {
+		throw new BadRequestError(
+			'Headers in a batch request must include a "content-type" header if they are provided',
+		);
+	}
+	if (
+		requests.find(
+			(request) =>
+				request.headers?.authorization != null ||
+				request.url?.includes('apikey='),
```

Comment:
Technically params for authentication can be customized so this isn't perfect - I'm not sure if it's something we can account for but worth being aware of